tests/ui-api-layer-acceptance-tests/graphql/config.go

```diff
@@ -51,11 +53,20 @@ func loadConfig() config {
 
 	if graphQlEndpoint == "" {
 
-		config.graphQlEndpoint = fmt.Sprintf("https://ui-api.%s/graphql", domain)
+		graphQlEndpoint = fmt.Sprintf("https://ui-api.%s/graphql", domain)
```

Comment:
Could you please install our "github.com/vrischmann/envconfig" library? There are too much envs, we need it 😛 Thanks!